app/lib/medical-transcription-analysis-stack.ts

```diff
@@ -366,8 +366,7 @@ export class MedicalTranscriptionAnalysisStack extends cdk.Stack {
         onEventAthenaLambda.addToRolePolicy(
           new iam.PolicyStatement({
             effect: iam.Effect.ALLOW,
-            actions: ["athena:StartQueryExecution", "athena:CreateNamedQuery", "athena:DeleteNamedQuery", "athena:GetQueryResults",
-            "athena:GetWorkGroup", "athena:CancelQueryExecution", "athena:StopQueryExecution", "athena:GetQueryExecution"],
+            actions: ["athena:*"],
```

Comment:
Do we need to assign "*"i.e. all permissions? We could make the permissions and resources more specific.

Reply:
Done. https://github.com/aws-samples/medical-transcription-analysis/blob/f05b763e09b17a207c02070a30bf2ee564f1d99d/app/lib/medical-transcription-analysis-stack.ts#L369-L370